lib/tasks/paperclip.rake

```diff
@@ -46,7 +46,7 @@ namespace :paperclip do
           attachment = instance.send(name)
           begin
             attachment.reprocess!(*styles)
-          rescue Exception => e
+          rescue => e
```

Comment:
I think it would be even better to explicitly say `rescue StandardError => e`

Reply:
Agreed, this appears to have the same problem as before, it's just now even broader. Agree in principle that rescuing from `StandardError` would be an improvement unless there's some system reason that we needed to use `Exception` here. Since it's not covered by a test I think we can assume this was just an oversight.